pkg/client/v1/nimbleos/access_control_record.go

```diff
@@ -76,3 +94,27 @@ type AccessControlRecord struct {
 	// AccessProtocol - Access protocol of the volume.
 	AccessProtocol *NsAccessProtocol `json:"access_protocol,omitempty"`
 }
+
+// Struct for AccessControlRecordFields
```

Comment:
I suggest:

// AccessControlRecordStringFields provides a string representation for each AccessControlRecord field.